sda/internal/database/database.go

```diff
@@ -48,6 +48,12 @@ type SyncData struct {
 	Checksum string
 }
 
+type SubmissionFileInfo struct {
+	InboxPath string `json:"inboxPath"`
+	Status    string `json:"fileStatus"`
+	CreateAt  string `json:"createAt"`
```

Comment:
I suspect it might also be helpful to provide the uploaded size (size the user saw before upload), and I don't see any problems with that (e.g. information leakage is not a thing since you only see your files).